README.md

```diff
@@ -386,44 +386,27 @@ In order to solve circular references between databases that may have been incor
 ```
 
 ## Packaging support
-`MSBuild.Sdk.SqlProj` supports packaging your project into a [NuGet](https://www.nuget.org) package using the `dotnet pack` command. In order for this to work, you'll need to add a `.nuspec` file next to your project file with the same name. For example, if your `.csproj` is called `TestProject.csproj` you'll need to add a `TestProject.nuspec` file in the same folder. Fill this file with the following contents and replace the placeholder with the appropriate value:
 
-```xml
-<?xml version="1.0" encoding="utf-8" ?>
-<package xmlns="http://schemas.microsoft.com/packaging/2011/10/nuspec.xsd">
-  <metadata>
-    <id>$id$</id>
-    <version>$version$</version>
-    <description>$description$</description>
-    <authors>$authors$</authors>
-    <owners>$authors$</owners>
-    <copyright>$copyright$</copyright>
-    <projectUrl>$projecturl$</projectUrl>
-    <tags>$tags$</tags>
-    <packageTypes>
-      <packageType name="$packagetype$" />
-    </packageTypes>
-    <repository type="git" url="<repository-url>" />
-  </metadata>
-  <files>
-    <file src="bin\$configuration$\$tfm$\*.dacpac" target="tools/" />
-  </files>
-</package>
-```
+`MSBuild.Sdk.SqlProj` version 2.8.0 and later supports packaging your project into a [NuGet](https://www.nuget.org) package using the `dotnet pack` command and does not require a `.nuspec` file.
```

Comment:
This is kind of a breaking change - should I leave the old docs section for now?

Reply:
I guess the real question is whether we would still like to support the ability to customize the `.nuspec` file or just stick with the automatically generated one.

Reply:
I vote for "just  stick with the automatically generated one" ...

Reply:
@jmezach Thoughts?

Reply:
@jmezach I just simplified the readme section based on the latest Sdk.Props changes

Reply:
Does it cost us anything to support the ability to customize the `.nuspec` file? If it's just a matter of leaving these docs in place, we may as well! If we think it will mean actual maintenance effort in future releases, then I'm fine with just supporting the automatically generated one.

Reply:
@jeffrosenberg It means additional testing (and maybe special casing) to confirm that both approaches are supported.

Reply:
Got it, I'm fine with this as-is then :slightly_smiling_face: